main.go

```diff
@@ -385,13 +378,23 @@ This command is meant to be used from the boot GRUB menu, but can be also starte
 			&cli.BoolFlag{
 				Name: "shell",
 			},
+			&cli.StringFlag{
```

Comment:
we can just declare this flag at the top and reuse it everywhere?

Reply:
done